### PR TITLE
feat(promitor-scraper): add extraVolumes and extraVolumeMounts

### LINKED ISSUE #81: Provide capability to add extra volumes & volume mounts agents
The current chart deployment does not contain the possibility to add extraVolumeMounts and extraVolumes values

### PR DESCRIPTION
**Pull request by isantospardo**
Relates to #81

### CODE REVIEW

**Review by tomkerkhove — COMMENTED** [review]
Thanks!

Would you mind:
- Updating the README with the new values, please?
- Updating our resource discovery agents as well, please?

File: promitor-agent-scraper/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-version: 2.6.2
-appVersion: 2.5.1
+version: 2.7.1
+appVersion: 2.6.1
```
Comment (tomkerkhove) [inline promitor-agent-scraper/Chart.yaml:3]: Can you revert this please?

**Comment by tomkerkhove** [comment]
Also, are you using Promitor in production? If so, am I allowed to list you/CERN as an end-user on promitor.io?

**Comment by isantospardo** [comment]
Hi @tomkerkhove I applied all the changes suggested. Also, I am not currectly working for CERN, so unfortunately no.

**Comment by tomkerkhove** [comment]
Ha I was asking because of this:
image


Thanks for the changes!

**Comment by eyenx** [comment]
> Also, are you using Promitor in production? If so, am I allowed to list you/CERN as an end-user on promitor.io?

Hi @tomkerkhove you can list @adfinis-sygroup as end-user.

**Comment by tomkerkhove** [comment]
Failing due to #78, merging - Thanks @isantospardo!

**Comment by tomkerkhove** [comment]
> > Also, are you using Promitor in production? If so, am I allowed to list you/CERN as an end-user on promitor.io?
> 
> Hi @tomkerkhove you can list @adfinis-sygroup as end-user.

Oh wow, thank you @eyenx! Are you willing to open a PR similar to https://github.com/promitor/promitor.io/pull/11 so that the correct logo is used or do you prefer me to do it?

**Comment by isantospardo** [comment]
MR opened in https://github.com/promitor/promitor.io/pull/16

**Pull request merged**